packages/react-next/src/common/_constants.scss

```diff
@@ -10,3 +10,5 @@ $animationDuration1: 0.167s;
 $animationDuration2: 0.267s;
 $animationDuration3: 0.367s;
 $animationDuration4: 0.467s;
+
+$CONTEXTUAL_MENU_ITEM_HEIGHT: 36px;
```

Comment:
camelCase?

Reply:
This needs to match CONTEXTUAL_MENU_ITEM_HEIGHT in ContextualMenu.cnstyles.ts. I'll add a comment to that effect.